pypolygames/tests/test_zoo.py

```diff
@@ -6,11 +6,42 @@
 from unittest import SkipTest
 import pytest
 import torch
+import torch.nn.functional as F
 from .. import model_zoo
 from ..model_zoo.utils import get_game_info
 from .. import params
 from .. import utils
 
+# NOTE: Copied this loss from a model in the first commit
```

Comment:
Bit of a strange test, but it will do . It has a bunch of asserts indeed.

Without a loss how are the NNs trained with this code base? Maybe we don't care because we're just going to have to write an adversarial training part anyways.

Reply:
There is a loss ([here](https://github.com/AlignmentResearch/polygames/blob/main/pypolygames/model_zoo/loss.py)) just it doesn't belong to the model, but rather to the ModelManager I believe.